vm/src/sysmodule.rs

```diff
@@ -222,6 +222,78 @@ fn sys_displayhook(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult<()> {
     Ok(())
 }
 
+#[pystruct_sequence(name = "sys._getwindowsversion_type")]
```

Comment:
Actual name below is required to be aliased with `_` prefix, but this one is not. This name can be `sys.getwindowsversion` without any problem.

To make things clear, 
- The name here is the type name.
- The name below is the attrname of the sys module.

Reply:
Oh, thank you! I was bit confused about the differences between them.